scripts/newinstall.sh

```diff
@@ -501,15 +501,24 @@ n8l::up2date_check() {
 	set +e
 
 	local amidiff
-	amidiff=$($CURL "$CURL_OPTS" -L "$NEWINSTALL_URL" | diff --brief - "$0")
+  diff --brief "$0" <($CURL "$CURL_OPTS" -L "$NEWINSTALL_URL") > /dev/null
```

Comment:
whitespace?

Reply:
Sorry, I corrected the missing tab. However I need some support for the ruby script.

Reply:
Sure, I'll take a look.

Reply:
Thank you very much